lib/player.js

```diff
@@ -37,7 +37,7 @@ Player.prototype = {
       if(distance < this.mass) {
         food.splice(i, 1);
         this.mass += 1;
-        if (this.speed > 0.8){ this.speed -= 0.03; }
+        if (this.speed > 1.0){ this.speed -= 0.003; }
```

Comment:
Good for now. Needs to be tweaked because, when playing, we noticed that at times a smaller circle is moving at the same speed of a much larger circle. hitting the minimum speed too quickly.